engine/samples/hello-cubos/page.md

```diff
@@ -0,0 +1,107 @@
+# Hello CUBOS. {#examples-engine-hello-cubos}
+
+@brief Using @ref cubos::engine::Cubos "Cubos" to create a simple program.
+
+This example shows the basics of how @ref cubos::engine::Cubos is used, by making a simple "Hello World" program.
+
+@snippet hello-cubos/main.cpp Include
+
+First we'll need to get a `Cubos` object.
+
+@snippet hello-cubos/main.cpp Engine
+
+The @ref cubos::engine::Cubos "Cubos" class represents the engine.
+We'll need it to add functionality to our program.
+
+Let's start by defining what functionality we want to add.
+
+@snippet hello-cubos/main.cpp Hello Cubos
+
+This function simply prints `Hello CUBOS` to the console.
+It uses one of CUBOS.'s logging macros.
+You can find more about them @ref core\include\cubos\core\log.hpp "here".
```

Comment:
This link is broken, perhaps because you're using \ instead of /

Reply:
I hate how they couldn't agree on which side to tilt the thingy to